packages/config/lib/store.js

```diff
@@ -86,7 +90,7 @@ class Store {
     }
 
     if (!app && type) {
-      const toLoad = `https://platformatic.dev/schemas/v${pltVersion}/${type}`
+      const toLoad = `https://schemas.platformatic.dev/@platformatic/${type}/${pltVersion}.json`
```

Comment:
We should keep supporting the _old_ schema as well.

Reply:
fixed